src/App.js

```diff
@@ -193,8 +193,8 @@ export class App extends Component{
               )
             }}
           />
+          <Footer/>
         </div>
-        <Footer/>
       </>
```

Comment:
I think with the footer back where it is we can get rid of the ReactFragment on 198